keyboards/faunchpad/config.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1337
+#define DEVICE_VER      1
+#define MANUFACTURER    g Heavy Industries
+#define PRODUCT     		FaunchPad
```

Comment:
Would you mind fixing all the mixed tabs/spaces in these files? :)

Reply:
Will do in next commit

Reply:
Fixed in c6d5906